charts/kubeai/values.yaml

```diff
@@ -20,6 +20,7 @@ modelServers:
       default: "vllm/vllm-openai:v0.6.3.post1"
       cpu: "substratusai/vllm:v0.6.3.post1-cpu"
       google-tpu: "substratusai/vllm:v0.6.3.post1-tpu"
+      gh200: "drikster80/vllm-gh200-openai:v0.6.3.post1"
```

Comment:
the plan would be to remove this once the default image works with `gh200`. I do like how you created an image just for gh200 for now.